frontend/benefit/handler/next.config.js

```diff
@@ -0,0 +1,42 @@
+const webpack = require('webpack');
```

Comment:
Perhaps this next.config.js is exactly the same for each project? This could be also moved to `frontend` root folder to avoid copy-paste?

Reply:
not exactly the same

Reply:
fixed